content/SUMMARY.md

```diff
@@ -28,6 +28,7 @@
 * [Basics](basics/index.md)
   * [HTML](basics/html.md)
   * [CSS](basics/css.md)
+  * [Sass](basics/sass.md)
```

Comment:
@dhsathiya I think this should be sub-chapter of CSS. What do you say?

Reply:
Sass is CSS pre-processor.